docs/content/preview/reference/configuration/default-ports.md

```diff
@@ -60,31 +60,35 @@ The following common ports are required for firewall rules:
 
 ## Prometheus monitoring
 
-YugabyteDB servers expose time-series performance metrics in the [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format) on multiple HTTP endpoints. These endpoints have the following structure.
+YugabyteDB servers expose time-series performance metrics in the [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format) on multiple HTTP endpoints that have the following structure:
 
 ```
 <target>/prometheus-metrics
 ```
 
-You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node level metrics.
+You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node-level metrics:
+
+| Service            | Port |
+| ------------------ | ---- |
+| Prometheus metrics | 9300 |
```

Comment:
`Node metrics` would be a better name for this than `prometheus metrics`. The metrics are ingested into prometheus but node exporter can be used without prometheus as well.

Reply:
Changed